### DISCUSSION TITLE
How-To Chia CLI Ubuntu farming no wallet

### DISCUSSION THREAD

**Discussion by jmz-bit** [Chia-Network/chia-blockchain#8644] [Support]
I'm using Ubuntu 20.04 and currently running Chia 1.2.7 with GUI installed from .deb file.

I'm farming to a cold wallet. I think because of that I don't need to keep wallet running. I also don't need Chia GUI as I'm using farmr app to monitor my farming process.

I would like to save computer resources (RAM and CPU).

Is it possible to run Chia in terminal without a wallet?

is this proper command to do what I need?

chia start farmer-no-wallet

**Comment by jmz-bit** [comment]
I tried command:

./chia start famer-no-wallet

and it works.

**Comment by BrandtH22** [comment]
@jmz-bit ,based on your previous comment it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
